curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -1218,6 +1243,85 @@ void ChunkCacheManager::ReadByReadCache(uint64_t chunkPos, uint64_t readLen,
     return;
 }
 
+void ChunkCacheManager::ReadByFlushData(uint64_t chunkPos, uint64_t readLen,
```

Comment:
the main logic of this function is very similar to the loop in `ReadByReadCache`, so can you union the logic of these two functions?

Reply:
yes，todo later